deploy/database/updates/00344_custom_recipe_02.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE game_player_map MODIFY original_recipe VARCHAR(200) ;
+ALTER TABLE game_player_map MODIFY alt_recipe VARCHAR(200) ;
```

Comment:
It's been awhile --- changing `game_player_map` means that when i do the site update, i need to rerun `views.game.sql`, right?

Reply:
I don't believe that this should change anything with the views, since these are just SELECTs at heart, and they don't care about the type of the variables.

Reply:
Seems like you're right.